README.md

```diff
@@ -141,35 +149,120 @@ In future iterations of the project, the Transaction Submission Service will pro
+
+> [!NOTE]  
+> In order to avoid concurrency issues, the SDP only supports one Event Broker or Background Jobs at a time.
+
+#### Kafka
+We recommend Kafka for organizations that require high throughput and low latency. Organizations that plan on hosting multiple tenants on the SDP should consider using Kafka.
+
+**1. Topics**
+
+* `events.receiver-wallets.new_invitation`: This topic is used to send disbursement invites to recipients. *[Producer: Core, Consumer: Core]*
+* `events.payment.ready_to_pay`: This topic is used to submit payments from the Core to the TSS. *[Producer: Core, Consumer: TSS]*
+* `events.payment.payment_completed`: This topic is used to notify the Core that a payment has been completed. *[Producer: TSS, Consumer: Core]*
+
+**2. Configuration**
+
+In order to use Kafka, you need to set the following environment variables for SDP and TSS. 
+
+```sh
+  EVENT_BROKER_TYPE: "KAFKA"
+  BROKER_URLS: # comma separated list of broker urls
+  CONSUMER_GROUP_ID: # consumer group id
+  KAFKA_SECURITY_PROTOCOL: # possible values "PLAINTEXT", "SASL_SSL", "SASL_PLAINTEXT" or "SSL"
+  KAFKA_SASL_USERNAME: # username for SASL authentication. Required if KAFKA_SECURITY_PROTOCOL is "SASL_SSL" or "SASL_PLAINTEXT"
+  KAFKA_SASL_PASSWORD: # password for SASL authentication. Required if KAFKA_SECURITY_PROTOCOL is "SASL_SSL" or "SASL_PLAINTEXT"
+  KAFKA_SSL_ACCESS_KEY: # access key (keystore) in PEM format. Required if KAFKA_SECURITY_PROTOCOL is "SSL" or "SASL_SSL"
+  KAFKA_SSL_ACCESS_CERTIFICATE: # certificate in PEM format that matches the access key. Required if KAFKA_SECURITY_PROTOCOL is "SSL" or "SASL_SSL"
+```
+
+#### Background Jobs
+We recommend Background Jobs for organizations that require a simpler setup and do not need high throughput or low latency. Organizations that plan on hosting a single tenant on the SDP should consider using Background Jobs.
+
+**1. Jobs**
+
+> [!NOTE]  
+> Certain jobs are not listed here because they cannot be configured and are necessary to the functioning of the SDP. 
+
+* `send_receiver_wallets_sms_invitation_job`: This job is used to send disbursement invites to recipients. Its interval is configured through the `SCHEDULER_RECEIVER_INVITATION_JOB_SECONDS` environment variable.
+* `payment_to_submitter_job`: This job is used to submit payments from Core to the TSS. Its interval is configured through the `SCHEDULER_PAYMENT_JOB_SECONDS` environment variable.
+* `payment_from_submitter_job`: This job is used to notify Core that a payment has been completed. Its interval is configured through the `SCHEDULER_PAYMENT_JOB_SECONDS` environment variable.
+* `patch_anchor_platform_transactions_completion`: This job is used to patch transactions in Anchor Platform once payments reach a final state. Its interval is configured through the `SCHEDULER_PAYMENT_JOB_SECONDS` environment variable.
```

Comment:
Perhaps we could add what "final state" means.

```suggestion
* `patch_anchor_platform_transactions_completion`: This job is used to patch transactions in Anchor Platform once payments reach a final state (`SUCCESS` or `FAILED`). Its interval is configured through the `SCHEDULER_PAYMENT_JOB_SECONDS` environment variable.
```

Reply:
addressed in 1b836460e43e6fc0fc48184e4626f9186ebec034